### PR TITLE
Added Docker files.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: web_scraping/gog_scraper/dockerise.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+source .env
```
Comment (MichaelHassan) [inline web_scraping/gog_scraper/dockerise.sh:1]: Nice useful script

File: web_scraping/gog_scraper/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Specifies latest image of python
+FROM public.ecr.aws/lambda/python:latest
+
+# Copies txt file containing env package requirements
+COPY requirements.txt .
```
Comment (Leon-JS02) [inline web_scraping/gog_scraper/Dockerfile:5]: Neat and small dockerfile.

File: web_scraping/gog_scraper/README.md [hunk 3]
```diff
@@ -14,10 +14,20 @@ Ensure that you have the following:
 
 ## 📂 Setup
 
-TBC
+1. Create a `.env` file with the following content:
+
```
Comment (Leon-JS02) [inline web_scraping/gog_scraper/README.md:18]: Informative changes to readme.

**Review by MichaelHassan — APPROVED** [review]
Looks good, and tested locally.

**Review by Leon-JS02 — APPROVED** [review]
Good dockerisation.

**Pull request merged**